back-end/routes/api.php

```diff
@@ -10,10 +11,24 @@
 |
 | Here is where you can register API routes for your application. These
 | routes are loaded by the RouteServiceProvider within a group which
-| is assigned the "api" middleware group. Enjoy building your API!
+| is assigned the "aTpi" middleware group. Enjoy building your API!
 |
 */
 
+Route::get('/v1', function () {
+    return response('You are at the version 1 of this api');
+});
+
+/*------------------User-Routes------------------*/
+Route::group(['prefix' => '/v1/users'], function () {
+    Route::post('/', [UsersController::class, 'store']);
+    Route::get('/', [UsersController::class, 'index']);
+    Route::get('/{user_id}', [UsersController::class, 'show']);
+    Route::put('/update/{user_id}', [UsersController::class, 'update']);
+    Route::delete('/delete/{user_id}', [UsersController::class, 'destroy']);
```

Comment:
```suggestion
    Route::put('/{user_id}', [UsersController::class, 'update']);
    Route::delete('/{user_id}', [UsersController::class, 'destroy']);
```
You can ommit those, since you can already differentiate a route by providing a method like `put`, `delete`, `get` even it has same api route.

And then for delete/put/get users routes, you may want to transfer those to protected routes (with sanctum middleware). Since we need to be authenticated to access those routes for security purposes.

Reply:
This is noted, Sir Chev.